.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: "go"
 sudo: false
 go:
+  - 1.16.x
   - 1.12.x
```

Comment:
Question: do you know what these older versions do for us? I hadn't thought about this section before

Reply:
I think it will run with other versions of GO on the repo.